d365fo.tools/internal/functions/start-lcsuploadv2.ps1

```diff
@@ -119,6 +120,13 @@ function Start-LcsUploadV2 {
     begin {
         Invoke-TimeSignal -Start
         
+        if ($Description) {
```

Comment:
:pick:

Thanks for the pr. Just some "nice to have" comment: It would be nicer to directly use the `$Description` parameter as value for `$FileDescription`. Unless there is some benefit to explicitely `$null`ing it?

Reply:
I agree. I was basing this on v1 of the same function but I agree that it doesn't make sense. I'll update the PR.